pkgs/by-name/ch/chatbox/package.nix

```
@@ -2,35 +2,40 @@
   lib,
   appimageTools,
   fetchurl,
+  writeScript,
 }:
-let
+
+appimageTools.wrapAppImage rec {
   pname = "chatbox";
-  version = "1.11.3";
+  version = "1.11.12";
 
-  src = fetchurl {
+  source = fetchurl {
     url = "https://download.chatboxai.app/releases/Chatbox-${version}-x86_64.AppImage";
-    hash = "sha256-YMZsgbCkG/cZd77DOVZj6jVJDzWMcrt92soxyLNr2QI=";
+    hash = "sha256-sfR4fXpREPdEoPoez5ugCpn/ubpB6EhkvpVq6rz0BSg=";
   };
 
-  appimageContents = appimageTools.extract { inherit pname version src; };
-in
-appimageTools.wrapType2 {
-  inherit pname version src;
+  src = appimageTools.extract {
+    inherit pname version;
+    src = source;
+  };
 
   extraInstallCommands = ''
-    install -m 444 -D ${appimageContents}/xyz.chatboxapp.app.desktop $out/share/applications/chatbox.desktop
-    install -m 444 -D ${appimageContents}/usr/share/icons/hicolor/512x512/apps/xyz.chatboxapp.app.png $out/share/icons/hicolor/512x512/apps/chatbox.png
+    install -m 444 -D ${src}/xyz.chatboxapp.app.desktop $out/share/applications/chatbox.desktop
+    install -m 444 -D ${src}/usr/share/icons/hicolor/512x512/apps/xyz.chatboxapp.app.png $out/share/icons/hicolor/512x512/apps/chatbox.png
     substituteInPlace $out/share/applications/chatbox.desktop \
       --replace-fail 'Exec=AppRun' 'Exec=chatbox' \
       --replace-fail 'Icon=xyz.chatboxapp.app' 'Icon=chatbox'
   '';
 
-  passthru.updateScript = ''
-    #!/usr/bin/env nix-shell
-    #!nix-shell -i bash -p curl gnugrep common-updater-scripts
-    version=$(curl -I -X GET https://chatboxai.app/install_chatbox/linux | grep -oP 'Chatbox-\K[0-9]+\.[0-9]+\.[0-9]+')
-    update-source-version chatbox $version
-  '';
+  passthru = {
+    src = source;
```

I could not find an example of this entry. Does this fix the problem of r-ryantm silently fails the update? [the log](https://r.ryantm.com/log/chatbox/2025-04-17.log)